src/Verifier/SAW/Simulator/RME.hs

```diff
@@ -212,6 +212,10 @@ prims =
   , Prims.bpIntLt  = pure2 (\x y -> RME.constant (x < y))
   , Prims.bpIntMin = undefined--pure2 min
   , Prims.bpIntMax = undefined--pure2 max
+    -- Array operations
+  , Prims.bpArrayConstant = error "bpArrayConstant unimplemented for backend"
+  , Prims.bpArrayLookup = error "bpArrayLookup unimplemented for backend"
+  , Prims.bpArrayUpdate = error "bpArrayUpdate unimplemented for backend"
```

Comment:
Could you clarify which backend this is here, too? And, actually, it might be better to have this message just say "array primitives aren't supported in the RME backend", as that would be more meaningful to a user.

Reply:
done